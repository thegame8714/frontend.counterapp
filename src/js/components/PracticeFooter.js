import React, { Component } from 'react'
import { Link } from 'react-router';

class PracticeFooter extends Component {
	constructor() {
		super();
		this.state = {
			practice: 'Start practice',
			schedule: 'Schedule',
			edit: 'Edit'
		}
	}
	render() {
		return (
			<div className="bar bar-tab">
				<Link className="tab-item" to="/start">
					<span className="icon icon-play"></span>
					<span className="tab-label">{this.state.practice}</span>
				</Link>
				<Link className="tab-item" to="/schedule">
					<span className="icon icon-compose"></span>
					<span className="tab-label">{this.state.schedule}</span>
				</Link>
				<Link className="tab-item" to="/edit">
					<span className="icon icon-edit"></span>
					<span className="tab-label">{this.state.edit}</span>
				</Link>
			</div>
		)
	}
}

export default PracticeFooter;
