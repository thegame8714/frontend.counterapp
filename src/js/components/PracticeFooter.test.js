import React from 'react';
import { mount } from 'enzyme';
import { mountToJson } from 'enzyme-to-json';
import PracticeFooter from './PracticeFooter';


describe('<PracticeFooter />', () => {
  it('should match snapshot', () => {
    const wrapper = mount(<PracticeFooter />);
    expect(mountToJson(wrapper)).toMatchSnapshot();
  });
});