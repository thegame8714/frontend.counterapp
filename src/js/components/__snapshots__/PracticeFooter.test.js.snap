exports[`<PracticeFooter /> should match snapshot 1`] = `
<PracticeFooter>
  <div
    className="bar bar-tab">
    <Link
      className="tab-item"
      onlyActiveOnIndex={false}
      style={Object {}}
      to="/start">
      <a
        className="tab-item"
        onClick={[Function]}
        style={Object {}}>
        <span
          className="icon icon-play" />
        <span
          className="tab-label">
          Start practice
        </span>
      </a>
    </Link>
    <Link
      className="tab-item"
      onlyActiveOnIndex={false}
      style={Object {}}
      to="/schedule">
      <a
        className="tab-item"
        onClick={[Function]}
        style={Object {}}>
        <span
          className="icon icon-compose" />
        <span
          className="tab-label">
          Schedule
        </span>
      </a>
    </Link>
    <Link
      className="tab-item"
      onlyActiveOnIndex={false}
      style={Object {}}
      to="/edit">
      <a
        className="tab-item"
        onClick={[Function]}
        style={Object {}}>
        <span
          className="icon icon-edit" />
        <span
          className="tab-label">
          Edit
        </span>
      </a>
    </Link>
  </div>
</PracticeFooter>
`;
